site/docs/components/grid-layout/index.mdx

```diff
@@ -1,14 +1,14 @@
 ---
-title: Grid Layout
+title: Grid layout
 data:
-  description: "Grid Layout manages the layout of the main content area in your application screen or webpage. It aligns content to a system of equally distributed columns and rows to create a cohesive and organized UI. When used with responsive breakpoints, Grid Layout can be used to determine how the same content is displayed across different viewport sizes or device screen sizes."
+  description: "`GridLayout` and `GridItem` manage the layout of the main content area in your application screen or webpage. It aligns content to a system of equally distributed columns and rows to create a cohesive and organized user interface. When combined with responsive breakpoints, `GridLayout` can determine the display of specific content across different viewport sizes or device screen sizes."
```

Comment:
`GridLayout` can determine the display of specific content across different viewport sizes or device screen sizes.

This makes it sound like this is an automatic behaviour but it needs to be configured